src/main/kotlin/com/faendir/intellij/gradleVersionCatalogs/kotlin/inlayhints/VersionCatalogInlayHintsProvider.kt

```diff
@@ -45,6 +45,7 @@ class VersionCatalogInlayHintsProvider : InlayHintsProvider<NoSettings> {
                                         VCElementType.LIBRARY -> resolvePotentiallyTabledDefinition(referencedValue, "module")
                                         VCElementType.VERSION -> referencedValue.text?.unquote()
                                         VCElementType.PLUGIN -> resolvePotentiallyTabledDefinition(referencedValue, "id")
+                                        VCElementType.BUNDLE -> resolveBundlesDefinition(referencedValue)
```

Comment:
Bundles can be very large, not sure if inlay hints are the best idea here. Maybe with cutoff and ellipsis after 3 or 5 or something?

Reply:
Yeah！you're right. I'll make a samll change.